cli/commands.js

```diff
@@ -72,11 +72,11 @@ const defaultCommands = {
         options: [
             {
                 option: '--fork',
-                desc: 'Run generators in parallel without any interaction. Files will be overwritten without prompt',
+                desc: 'Run generators in parallel without any interaction. Files will be overwritten without prompt. This is default for new applications',
             },
             {
                 option: '--interactive',
-                desc: 'Run generation in series so that questions can be interacted with. This is the default',
+                desc: 'Run generation in series so that questions can be interacted with. This is the default when there is an existing application in the folder',
```

Comment:
```suggestion
                desc: `Generate multiple applications in series so that questions can be interacted with.
  This is the default when there is an existing application configuration in some of the applications folders`,
```